### PR TITLE
[INTER-544] Revisit

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

**Review by Orkuncakilkaya — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,41 +1,18 @@
...
\ No newline at end of file
+    name: 'Publish new version'
+    uses: fingerprintjs/dx-team-toolkit/.github/workflows/release-server-sdk.yml@v1
+    with:
+      language: php
+      language-version: '8.3'
+      prepare-command: |
+        composer install -q --profile --ignore-platform-reqs --no-interaction --no-ansi --no-scripts --no-suggest --prefer-dist
+    secrets:
+      GH_RELEASE_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
```
Comment (ilfa) [inline .github/workflows/release.yml:18]: We need to migrate to GH App

**Review by ilfa — CHANGES_REQUESTED** [review]
Update please functional tests to use reusable workflow to report test status and to do it in one message instead of how is it configured now (separate message for each version).

File: .github/workflows/release.yml (hunk 1 above)
Comment (TheUnderScorer) [reply .github/workflows/release.yml]: Done!

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        php_version: [ "7.3", "7.4", "8.0", "8.1", "8.2" ]
+        php_version: [ "7.3", "7.4", "8.0", "8.1", "8.2", "8.3" ]
 
     steps:
       - uses: actions/checkout@v2
```
Comment (ilfa) [inline .github/workflows/test.yml:18]: Here is still an old version.

**Review by ilfa — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (TheUnderScorer) [reply .github/workflows/test.yml]: Whoops, fixed!

**Review by ilfa — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-    </a>
+    <picture>
+     <source media="(prefers-color-scheme: dark)" srcset="https://fingerprintjs.github.io/home/resources/logo_light.svg" />
+     <source media="(prefers-color-scheme: light)" srcset="https://fingerprintjs.github.io/home/resources/logo_dark.svg" />
+     <img src="https://raw.githubusercontent.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/main/res/logo_dark.svg" alt="Fingerprint logo" width="312px" />
+   </picture>
+  </a>
 </p>
 <p align="center">
     <a href="https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/actions/workflows/release.yml">
```
Comment (ilfa) [inline README.md:11]: We need to fix formatting issues
image

**Review by ilfa — CHANGES_REQUESTED** [review]

File: .github/workflows/functional.yml [hunk 4]
```diff
@@ -36,11 +36,13 @@ jobs:
...
-        if: always()
-        uses: ravsamhq/notify-slack-action@0d9c6ff1de9903da88d24c0564f6e83cb28faca9
-        with:
-          status: ${{ job.status }}
-          notification_title: 'PHP ${{ matrix.php_version }} SDK Functional Test has {status_message}'
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  report-status:
+    needs: tests
```
Comment (ilfa) [inline .github/workflows/functional.yml:41]:
```suggestion
    needs: functional_tests
```

File: .github/workflows/functional.yml [hunk 5]
```diff
@@ -36,11 +36,13 @@ jobs:
...
+    if: always()
+    uses: fingerprintjs/dx-team-toolkit/.github/workflows/report-workflow-status.yml@v1
+    with:
+      notification_title: 'PHP SDK Tests has {status_message}'
+      job_status: ${{ needs.tests.result }}
```
Comment (ilfa) [inline .github/workflows/functional.yml:46]:
```suggestion
      job_status: ${{ needs.functional_tests.result }}
```

**Review by ilfa — CHANGES_REQUESTED** [review]

**Review by ilfa — APPROVED** [review]

**Pull request merged**